packages/buckets/src/api/index.ts

```diff
@@ -1,16 +1,18 @@
 import { grpc } from '@improbable-eng/grpc-web'
 import {
+  Archive as PbArchive,
   ArchiveConfig as ProtoArchiveConfig,
-  ArchiveInfoRequest,
-  ArchiveInfoResponse,
   ArchiveRenew as ProtoArchiveRenew,
   ArchiveRequest,
-  ArchiveStatusRequest,
-  ArchiveStatusResponse,
+  ArchivesRequest,
+  ArchivesResponse,
+  ArchiveStatus as PbArchiveStatus,
```

Comment:
We should probably just do something consistent here across the board at some point. @andrewxhill was using `_` prefixes: https://github.com/textileio/js-textile/blob/master/packages/buckets/src/api/index.ts#L24. I also like `pb_`... no real preference though. But something consistent would be good.